Build/build.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+git submodule sync
```

Comment:
In theory this needs to be done once only after the url was updated but it doesn't hurt to do it on every build.